website/docs/docs/collaborate/access-query-page.md

```diff
@@ -0,0 +1,150 @@
+
+### View details
+To view the details of the query, click on the **Details** tab to see:
+- Query metadata: Title, description, supplied SQL, compiled SQL
+- Connection details: Warehouse, database, schema, role, user
+- Query details: Query duration, status, columns, rows
+
+<Lightbox src="/img/docs/collaborate/query-page/qp-details.jpg" width="95%" title="Query page Details tab" />
+
+### Chart results
+
+The Query page also supports charting results. An analyst can:
+- Select the chart type and columns to visualize.
+- Choose from **line, bar, or dot charts**.
+- Charts work best with numerical or categorical data.
+
+<Lightbox src="/img/docs/collaborate/query-page/qp-chart.jpg" width="95%" title="Query page Chart tab" />
+
+### Query history
+
+The Query page also supports query history. The analyst can view the history of queries and their statuses (like Success, Error, Pending), and select a query to re-run and view the results.
+
+| Feature           | Availability |
+|------------------|--------------|
+| View or search for past queries | ✅ Yes |
+| Filter by status  | ✅ Yes (Success, Error, Pending) |
+| Export query history | ❌ No |
+
+How long is history stored?
+
+<Lightbox src="/img/docs/collaborate/query-page/qp-query-history.png" width="95%" title="Query page Query history icon" />
+
+<!-- add copilot section here if available 
+
+## Query with dbt Copilot
+
+dbt Cloud Copilot is a feature that allows you to query your data with natural language. It is available in the Query page....
+-->
+
+## Use dbt Explorer
+
+View dbt Explorer directly in the Query page to access your project lineage and project resources with access to tables, columns, metrics, and dimensions, and more — all integrated in the Query page interface. 
+
+This integrated view allows you to maintain your query workflow, while getting more context on models, semantic models, metrics, macros, and more. The integrated Explorer view provides you with:
+- Same search capabilities as Explorer
+- Allows you to narrow down displayed objects by type
+
+To access dbt Explorer, click on the **Explorer** icon on the top right of the Query page.
+
+<Lightbox src="/img/docs/collaborate/query-page/qp-explorer.png" width="90%" title="Query page integrated with dbt Explorer" />
+ 
+Questions:
+- Do we need a job run? 
+- Are the following available in the private beta?
+  - Automatic highlighting: Selecting an object in the console shows what?
+
+## Considerations 
+- Currently, there’s a limit of 500 returned queries (how to bypass or not possible?). Support for more rows coming soon! 
+- Although not currently supported, you will have the ability to save, organize, and share frequently used queries – allowing you to promote consistent use across teams without leaving the Query page.
+- Query page is available in development environment only, with support for multiple environments coming soon! 
+- Any access considerations?
```

Comment:
I am honestly not positive -- does this refer to RBAC in dbt Cloud or access to resources in the warehouse or some special third thing?